bin/q

```diff
@@ -778,7 +781,7 @@ class MaterializedFileState(object):
         if self.encoding == 'utf-8-sig' and self.lines_read == 0 and not self.skipped_bom:
             try:
                 BOM = self.f.read(3)
-                if BOM != '\xef\xbb\xbf':
+                if BOM != six.b('\xef\xbb\xbf'):
```

Comment:
b'\xef\xbb\xbf' works.

Reply:
saw your important note regarding six.b and six.u not being used anymore. Figured that I'll change everything to be consistent, and then do a find-replace sweep and change all of those to the regular b and u.